### ISSUE TITLE
Add methods to get invocations count of a mock service

### ISSUE THREAD

**Issue by lbroudoux** [microcks/microcks-testcontainers-node#83]
### Reason/Context

As of today, we lack of methods on `MicrocksContainer` class that could return the invocations count of a service mock.
This data is available in admin console, and is accessible via Metrics resource in REST API. So we would like to have this data easily available, we think it could be benefit for many.

And why do we need to check this invocation count ?

To be sure the mock as been invoked by the application we are covering with integration tests.
Without having this invocations count, we are not able to know whether our application had the correct behaviour or not.

### Description

In summary : it would be like what we do when we call the "verify" method of "Mockito".

**Comment by lbroudoux** [comment]
Now done.
